src/stan/io/reader.hpp

```diff
@@ -1018,15 +1108,15 @@ class reader {
    * dimensionality, reading from an unconstrained vector of the
    * appropriate size.
    *
-   * @param M Rows of Cholesky factor
-   * @param N Columns of Cholesky factor
+   * @param n Rows of Cholesky factor
```

Comment:
Nooooo!  Always m x n!  I'm going to stop writing this, but this should be put back to the conventional order everywhere.  If you want to be more verbose, Eigen uses `rows` and `cols`.